Issues building 0.3.5 with CMake on Linux with USE_OPENMP

Attempting to build OpenBLAS for the first time for repackaging, and configuring with the following options on Scientific Linux 7 with GCC 8.2.0:
```
cmake -DCMAKE_BUILD_TYPE=Release -DCMAKE_INSTALL_PREFIX=<install-dir> -DCMAKE_C_COMPILER=gcc -DCMAKE_Fortran_COMPILER=gfortran -DCMAKE_CXX_STANDARD=17 -DCMAKE_Fortran_FLAGS=-ffree-line-length-none -DBUILD_SHARED_LIBS=ON -DDYNAMIC_ARCH=ON -DDYNAMIC_OLDER=ON -DUSE_OPENMP=ON -DUSE_THREAD=ON -DCMAKE_POLICY_DEFAULT_CMP0054=NEW <src-dir>
```

[ The policy statement is necessary with newer CMake to avoid a warning about line 542 of `kernel/CMake`, which says `set(TARGET_CORE ${CORE})`. In order to be unamiguous, this line should be `set(TARGET_CORE "${CORE}")` ]

The build proceeds well until the final link, when everything fails with (_e.g._):
```
[100%] Linking Fortran executable cblat3
../lib/libopenblas.so.0.3: undefined reference to `zgetrf_parallel'
../lib/libopenblas.so.0.3: undefined reference to `cgetrf_parallel'
../lib/libopenblas.so.0.3: undefined reference to `sgetrf_parallel'
../lib/libopenblas.so.0.3: undefined reference to `dgetrf_parallel'
```

Indeed, these files appear never to be compiled?

Is there an easy way to resolve this issue with a workaround? There is a reason we'd prefer to stay with CMake if at all possible.

Thanks for your help,
Chris.

--- a/lapack/CMakeLists.txt
+++ b/lapack/CMakeLists.txt
@@ -63,7 +63,6 @@ if (USE_THREAD)
 
   # these do not have 'z' versions
   set(PARALLEL_SOURCES
-    ${GETRF_SRC}
     lauum/lauum_U_parallel.c
     lauum/lauum_L_parallel.c
     potrf/potrf_U_parallel.c
@@ -80,6 +79,10 @@ if (USE_THREAD)
     trtri/trtri_U_parallel.c
     trtri/trtri_L_parallel.c
   )
+
+  foreach (float_type ${FLOAT_TYPES})
+    GenerateNamedObjects("${GETRF_SRC}" "" "getrf_parallel" false "" "" false ${float_type})  
+  endforeach()
 
   GenerateNamedObjects("${PARALLEL_SOURCES}")
 endif ()

